src/Service.Tests/SqlTests/GraphQLMutationTests/GraphQLMutationTestBase.cs

```diff
@@ -40,6 +41,64 @@ public async Task InsertMutation(string dbQuery)
             SqlTestHelper.PerformTestEqualJsonStrings(expected, actual.ToString());
         }
 
+        /// <summary>
+        /// <code>Do: </code> Inserts new row and return all its columns
+        /// <code>Check: </code> A row is inserted in the table that has rows with default values as built_in methods.
+        /// it should insert it correctly with default values correctly handled by database.
+        /// </summary>
+        public async Task InsertMutationWithDefaultBuiltInFunctions(string dbQuery)
+        {
+            string graphQLMutationName = "createDefaultBuiltInFunction";
+            string graphQLMutation = @"
+                mutation {
+                    createDefaultBuiltInFunction(item: { user_value: 1234 }) {
+                        id
+                        user_value
+                        current_date
+                        current_utc_date
+                        random_number
+                        start_of_day
+                        end_of_day
+                        default_string_with_paranthesis
+                        default_function_string_with_paranthesis
+                        default_integer
+                    }
+                }
+            ";
+
+            JsonElement actual = await ExecuteGraphQLRequestAsync(graphQLMutation, graphQLMutationName, isAuthenticated: true);
+            string expected = await GetDatabaseResultAsync(dbQuery);
+            SqlTestHelper.PerformTestEqualJsonStrings(expected, actual.ToString());
+
+            // Assert the values
+            Assert.AreEqual(1234, actual.GetProperty("user_value").GetInt32());
```

Comment:
How is this specific check relevant to what you're testing?

Reply:
makes sense, updated the test.